Makefile

```diff
@@ -360,10 +370,15 @@ push-latest:
 .PHONY: push-binary-edge
 push-binary-edge:
 	aws s3 cp $(RELEASE_DIR)/opa_darwin_$(GOARCH) s3://$(S3_RELEASE_BUCKET)/edge/opa_darwin_$(GOARCH)
+	aws s3 cp $(RELEASE_DIR)/opa_darwin_$(GOARCH).sha256 s3://$(S3_RELEASE_BUCKET)/edge/opa_darwin_$(GOARCH).sha256
 	aws s3 cp $(RELEASE_DIR)/opa_darwin_arm64_static s3://$(S3_RELEASE_BUCKET)/edge/opa_darwin_arm64_static
+	aws s3 cp $(RELEASE_DIR)/opa_darwin_arm64_static.sha256 s3://$(S3_RELEASE_BUCKET)/edge/opa_darwin_arm64_static.sha256
 	aws s3 cp $(RELEASE_DIR)/opa_windows_$(GOARCH).exe s3://$(S3_RELEASE_BUCKET)/edge/opa_windows_$(GOARCH).exe
+	aws s3 cp $(RELEASE_DIR)/opa_windows_$(GOARCH).exe.sha256 s3://$(S3_RELEASE_BUCKET)/edge/opa_windows_$(GOARCH).exe.sha256
 	aws s3 cp $(RELEASE_DIR)/opa_linux_$(GOARCH) s3://$(S3_RELEASE_BUCKET)/edge/opa_linux_$(GOARCH)
+	aws s3 cp $(RELEASE_DIR)/opa_linux_$(GOARCH).sha256 s3://$(S3_RELEASE_BUCKET)/edge/opa_linux_$(GOARCH).sha256
 	aws s3 cp $(RELEASE_DIR)/opa_linux_$(GOARCH)_static s3://$(S3_RELEASE_BUCKET)/edge/opa_linux_$(GOARCH)_static
+	aws s3 cp $(RELEASE_DIR)/opa_linux_$(GOARCH)_static.sha256 s3://$(S3_RELEASE_BUCKET)/edge/opa_linux_$(GOARCH)_static.sha256
 
```

Comment:
💭 This is getting out of hand. But it seems you cannot provide multiple files in the first argument position. Perhaps a `aws sync` command?

```
aws s3 sync $(RELEASE_DIR)/ s3://$(S3_RELEASE_BUCKET)/edge
```
⚠️ untested ⚠️

Reply:
It's OK if you want to leave this as-is; I can take care of it in a follow-up. Enough moving parts for one PR.

Reply:
Sync is awesome, I can change to that. How about adding `--delete` too, it would remove any remote files that don't exist "locally"?

Reply:
Added `--delete`